buildconfig/pygame-stubs/transform.pyi

```diff
@@ -28,7 +28,7 @@ def average_surfaces(
     dest_surface: Optional[Surface] = None,
     palette_colors: Union[bool, int] = 1,
 ) -> Surface: ...
-def average_color(surface: Surface, rect: Optional[_RectValue] = None) -> Color: ...
+def average_color(surface: Surface, rect: Optional[_RectValue] = None, consider_alpha: Optional[bool] = False) -> Color: ...
```

Comment:
```suggestion
def average_color(surface: Surface, rect: Optional[_RectValue] = None, consider_alpha: bool = False) -> Color: ...
```
According to the [typing docs](https://docs.python.org/3/library/typing.html#typing.Optional) for the `Optional` typehint
> Optional type.
> `Optional[X]` is equivalent to `X | None` (or `Union[X, None]`).
> Note that this is not the same concept as an optional argument, which is one that has a default. An optional argument with a default does not require the `Optional` qualifier on its type annotation just because it is optional.

So here the argument is optional, but cannot take `None`. So `Optional` must not be used here

Reply:
Thanks for the review. Must have overlooked this in the documentation but that is the way to learn things. I haven't written much type hints so any hints are welcome.